specs/core/1_custody-game.md

```diff
@@ -274,22 +293,22 @@ def process_custody_reveal(state: BeaconState,
                            reveal: CustodyKeyReveal) -> None:
     assert verify_custody_key(state, reveal)
     revealer = state.validator_registry[reveal.revealer_index]
-    current_custody_period = epoch_to_custody_period(get_current_epoch(state))
+    validators_current_custody_reveal_period = get_validators_current_custody_reveal_period(state, reveal.revealer_index)
```

Comment:
```suggestion
    validators_next_custody_reveal_period = validator.latest_custody_reveal_period + 1
```
I think this needs to be changed to be consistent with having more time to reveal the secret